packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -7,6 +7,7 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/services.dart';
 
+import '../../widgets.dart';
```

Comment:
Please import what you need from within the widget folder directly instead of importing all of widget.dart.